openebs/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
2016/2017

Reply:
Sorry, my bad! It should be 2017! Updating it.